tools/delete_image_from_registry.bash

```diff
@@ -0,0 +1,27 @@
+# helper tool to delete tags from a docker registy
+# Need to run garbage collection afterwards in oder to free disk space https://docs.docker.com/registry/garbage-collection/
+# $> docker exec registry bin/registry garbage-collect --dry-run /etc/docker/registry/config.yml
+
+#https://stackoverflow.com/questions/25436742/how-to-delete-images-from-a-private-docker-registry
+if [$# -ne 4]; then
+ echo "illegal number of parameters"
+ echo use: RegistryUsername registryURL imagename tag
+fi
+
+USER=$1
+REGISTRY=$2
+NAME=$3
+TAG=$4
+read -sp 'registry password:' PASSWD 
```

Comment:
It's recommended to use the `-r` flag on the read call. When the flag is not set, backslashes are not read as normal characters but rather used for escaping the following sequence. A password usually doesn't have control characters, therefore it's probably better to get the "raw" input.

See also this shellcheck entry: https://github.com/koalaman/shellcheck/wiki/SC2162